package.json

```diff
@@ -33,7 +33,7 @@
         "graphql-request": "^7.1.0",
         "nuxt-svgo": "^4.0.2",
         "pinia": "^2.2.0",
-        "vue": "^3.4.35",
+        "vue": "^3.5.13",
```

Comment:
Let's avoid making changes to packages in PRs that aren't specifically focused on upgrading them. 🙏🏾